### PR TITLE
appboy-web-sdk v3 support

### PR DESCRIPTION
**Pull request by mericsson**
Merging https://github.com/segmentio/analytics.js-integrations/pull/528 submitted by @davidbielik

After merge
- [ ] I will release just to @davidbielik sources and he can confirm it is working as expected.
- [ ] Then, release to all customers on Jan 4 week.

https://segment.atlassian.net/browse/CC-7116
https://segment.atlassian.net/browse/STRATCONN-532

---

**What does this PR do?**

Adds support for version `3.1` in the `appboy` (braze) Web SDK integration.

Existing customers on older options (`1`, `2`, `2.7`) will continue to load those respective SDK versions. Customers who choose the new `3.1` option will load the `3.1` version of Braze SDK.

**Are there breaking changes in this PR?**

No

**Testing**

Added a new test for `initializeV3`

**`yarn test`**
image

**Any background context you want to provide?**

**Is there parity with the server-side/android/iOS integration components (if applicable)?**
n/a

**Does this require a new integration setting? If so, please explain how the new setting works**
Yes, a new version in the dropdown for `3.1`

**Links to helpful docs and other external resources**

https://github.com/Appboy/appboy-web-sdk/blob/master/CHANGELOG.md#300

### CODE REVIEW

File: integrations/appboy/lib/index.js [hunk 1]
```diff
@@ -47,6 +47,10 @@ var Appboy = (module.exports = integration('Appboy')
   .tag(
     'v2.7',
     '<script src="https://js.appboycdn.com/web-sdk/2.7/appboy.min.js">'
+  )
+  .tag(
+    'v3.1',
```
Comment (mericsson) [inline integrations/appboy/lib/index.js:52]: Added new 3.1 option to settings

image

**Review by briemcnally — APPROVED** [review]
LGTM

**Comment by mericsson** [comment]
@davidbielik this has been released to your test source -- let me know if it looks good and no regressions. If all looks good, it is expected to release in a tiered fashion starting Jan 4 and finishing Jan 7.

**Comment by mericsson** [comment]
@davidbielik Checking in if you had a chance to do testing yet. Understand it is holidays so may not have gotten to it. Let me know and after your approval I will queue for release. No rush from our end.

**Comment by davidbielik** [comment]
Hi @mericsson I'll DM you on slack, having trouble getting the new JS version in our account.

**Pull request merged**